indico/modules/events/registration/clone.py

```diff
@@ -141,6 +143,15 @@ def _clone_registrations(self, old_form, new_form, field_data_map):
                 if old_registration_data.storage_file_id is not None:
                     with old_registration_data.open() as fd:
                         new_registration_data.save(fd)
+                # Assigns the mapped session blocks for cloned event to the cloned registration.
+                if new_registration_data.field_data.field.input_type == 'sessions':
+                    mapped_sessions_data = []
+                    # If timetable is not cloned, assign an empty array to this field.
+                    if session_blocks_map:
+                        mapped_sessions_data = [next((session_blocks_map.get(session).id
```

Comment:
`somedict.get(foo).something` is always incorrect: if the dict key does not exist, get returns None, and `None.something` causes an exception. So use `session-blocks_map[session].id` instead.